src/pages/User.tsx

```diff
@@ -150,25 +189,43 @@ const UserPage = observer(() => {
         );
       })}
 
-      {userVestingContracts.length && (
-        <TitleRow>
-          <h2>Vesting Contracts</h2>
-        </TitleRow>
+      <TitleRow>
+        <h2>Vesting Contracts</h2>
+      </TitleRow>
+
+      {tokenVestingContracts.length ? (
+        tokenVestingContracts.map((contract, i, arr) => {
+          return (
+            <ListRow
+              clickable
+              key={contract.address}
+              borderBottom={i < arr.length - 1}
+              onClick={() => {
+                if (!contract.address) return;

```

Comment:
Lets move this into a function outside the onclick

Reply:
Done! Thanks